src/pages/articles/index.tsx

```diff
@@ -122,27 +124,58 @@ export default function ArticleArchive({
             fetchMoreArticles();
           }}
           largeFirstItem={showFirstItemLarge}
-          createLargeCard={(item) => (
-            <LargeCard
-              key={`lg-card-${item?.id}`}
-              {...getArticlePageCardProps(
-                item as ArticleType,
-                getRoutedInternalHref
-              )}
-            />
-          )}
-          createCard={(item) => (
-            <Card
-              key={`sm-card-${item?.id}`}
-              {...{
-                ...getArticlePageCardProps(
+          createLargeCard={(item) => {
+            const cardItem = item as ArticleType;
+            const itemCategories = cardItem?.categories;
+            return (
+              <LargeCard
+                key={`lg-card-${item?.id}`}
+                {...getArticlePageCardProps(
                   item as ArticleType,
                   getRoutedInternalHref
-                ),
-                text: '', // A design decision: The text is not wanted in the small cards
-              }}
-            />
-          )}
+                )}
+                //todo: fix any type
+                customContent={
+                  <ArticleDetails
+                    keywords={
+                      itemCategories?.edges
+                        ?.filter((category: any) => category?.node?.name)
+                        .map((category: any) => category?.node?.name || '') ||
+                      []
+                    }
+                  />
+                }
+              />
+            );
+          }}
+          createCard={(item) => {
+            const cardItem = item as ArticleType;
+            const itemCategories = cardItem?.categories;
+            return (
+              <Card
+                key={`sm-card-${item?.id}`}
+                {...{
+                  ...getArticlePageCardProps(
+                    item as ArticleType,
+                    getRoutedInternalHref
+                  ),
+
+                  text: '', // A design decision: The text is not wanted in the small cards
+                }}
+                //todo: fix any type
+                customContent={
+                  <ArticleDetails
+                    keywords={
+                      itemCategories?.edges
+                        ?.filter((category: any) => category?.node?.name)
+                        .map((category: any) => category?.node?.name || '') ||
+                      []
+                    }
+                  />
```

Comment:
Different version of RCHC-lib seems to automatically fix this. I tried `"react-helsinki-headless-cms": "1.0.0-alpha10-canary-f50fd90",` and it was good

Reply:
we have to check that